.eslintrc

```diff
@@ -1,53 +1,29 @@
 {
   "root": true,
   "parser": "@typescript-eslint/parser",
-  "plugins": [
-    "import",
-    "prettier",
-    "react-hooks",
-    "@typescript-eslint"
-  ],
+  "plugins": ["import", "prettier", "react-hooks", "@typescript-eslint"],
   "extends": [
     "prettier",
     "eslint:recommended",
     "plugin:@typescript-eslint/eslint-recommended",
     "plugin:@typescript-eslint/recommended"
   ],
   "rules": {
-    "arrow-body-style": [
-      "error",
-      "as-needed"
-    ],
+    "arrow-body-style": ["error", "as-needed"],
     "react/react-in-jsx-scope": "off",
     "no-trailing-spaces": "error",
     "react/no-children-prop": "off",
     "no-console": 1,
-    "quotes": [
-      "error",
-      "single"
-    ],
+    "quotes": ["error", "single"],
     "import/order": [
       "error",
       {
-        "groups": [
-          [
-            "builtin",
-            "external"
-          ],
-          [
-            "internal"
-          ],
-          [
-            "index",
-            "parent",
-            "sibling"
-          ]
-        ]
+        "groups": [["builtin", "external"], ["internal"], ["index", "parent", "sibling"]]
       }
     ],
     "import/no-cycle": "error",
     "react-hooks/rules-of-hooks": "error",
-    "react-hooks/exhaustive-deps": "warn",
```

Comment:
Maybe its better to leave this warning and to write eslint-ignore exhaustive deps if it is needed?